ecllibrary/teststd/Date/TestDate.ecl

```diff
@@ -237,6 +241,18 @@ EXPORT TestDate := MODULE
 
     ASSERT(Date.SecondsFromDateTimeRec(vCreateDateTime) = 917872496);
 
+    ASSERT(EXISTS(Date.TimeZone.UniqueTZAbbreviations()));
+    ASSERT(EXISTS(Date.TimeZone.TZDataForLocation('NORTH AMERICA')));
```

Comment:
Better to have a slightly better test - e.g., count the number of military time zones = 25
(I never knew about half these timezones before this PR!)